code/modules/cargo/bounties/assistant.dm

```diff
@@ -0,0 +1,270 @@
+	reward = 2000
+	required_count = 6
+	wanted_types = list(/obj/item/storage/toolbox)
+
+/datum/bounty/item/assistant/statue
+	name = "Statue"
+	description = "Central Command would like to commision an artsy statue for the lobby. Ship one out, when possible."
+	reward = 2000
+	wanted_types = list(/obj/structure/statue)
+
+/datum/bounty/item/assistant/clown_box
+	name = "Clown Box"
+	description = "The universe needs laughter. Stamp cardboard with a clown stamp and ship it out."
+	reward = 1800
+	wanted_types = list(/obj/item/storage/box/clown)
+
+/datum/bounty/item/assistant/cheesiehonkers
+	name = "Cheesie Honkers"
+	description = "Apparently the company that makes Cheesie Honkers is going out of business soon. CentCom wants to stock up before it happens!"
+	reward = 1200
+	required_count = 3
+	wanted_types = list(/obj/item/reagent_containers/food/snacks/cheesiehonkers)
+
+/datum/bounty/item/assistant/baseball_bat
+	name = "Baseball Bat"
+	description = "Baseball fever is going on at CentCom! Be a dear and ship them some baseball bats, so that management can live out their childhood dream."
+	reward = 2000
+	required_count = 5
+	wanted_types = list(/obj/item/melee/baseball_bat)
+
+/datum/bounty/item/assistant/extendohand
+	name = "Extendo-Hand"
+	description = "Commander Betsy is getting old, and can't bend over to get the telescreen remote anymore. Management has requested an extendo-hand to help her out."
+	reward = 2500
+	wanted_types = list(/obj/item/extendohand)
+
+/datum/bounty/item/assistant/donut
+	name = "Donuts"
+	description = "CentCom's security forces are facing heavy losses against the Syndicate. Ship donuts to raise morale."
+	reward = 3000
+	required_count = 10
+	wanted_types = list(/obj/item/reagent_containers/food/snacks/donut)
+
+/datum/bounty/item/assistant/donkpocket
+	name = "Donk-Pockets"
+	description = "Consumer safety recall: Warning. Donk-Pockets manufactured in the past year contain hazardous lizard biomatter. Return units to CentCom immediately."
+	reward = 3000
+	required_count = 10
+	wanted_types = list(/obj/item/reagent_containers/food/snacks/donkpocket)
+
+/datum/bounty/item/assistant/briefcase
+	name = "Briefcase"
+	description = "Central Command will be holding a business convention this year. Ship a few briefcases in support."
+	reward = 2500
+	required_count = 5
+	wanted_types = list(/obj/item/storage/briefcase)
+
+/datum/bounty/item/assistant/sunglasses
+	name = "Sunglasses"
+	description = "A famous blues duo is passing through the sector, but they've lost their shades and they can't perform. Ship new sunglasses to CentCom to rectify this."
+	reward = 3500
+	required_count = 2
+	wanted_types = list(/obj/item/clothing/glasses/sunglasses)
+
+/datum/bounty/item/assistant/gondola_hide
+	name = "Gondola Hide"
+	description = "Central Command has recently learned of strange creatures called Gondalas. If you catch one, ship its hide back to CentCom."
```

Comment:
it literally says Gondola in the name how is this misspelled here

Reply:
CentCom is like your old grandad